pilot/pkg/serviceregistry/kube/conversion_test.go

```diff
@@ -44,11 +44,11 @@ func TestConvertProtocol(t *testing.T) {
 		out   protocol.Instance
 	}
 	protocols := []protocolCase{
-		{"", coreV1.ProtocolTCP, protocol.TCP},
+		{"", coreV1.ProtocolTCP, protocol.Unsupported},
```

Comment:
Can you add tests for well known ports as well

Reply:
Sure. Tests added.